code/modules/lighting/lighting_overlay.dm

```diff
@@ -60,6 +60,14 @@
 		needs_update = 1
 		lighting_update_overlays += src
 
+/atom/movable/lighting_overlay/proc/update_shadowling_overlay(var/turf/T)
+	if(T.shadowling_bright_overlay)
```

Comment:
If/when PJB releases his goon-lights based system, lighting doesn't work the same, based on a fairly different concept. This code won't work then, and I don't want to have anything to worry about maintaining with it. Entirely my own subjective thing with this, the code will work with the current system.